app/views/posts/index.html.erb

```diff
@@ -1,5 +1,11 @@
 <% @posts.each do |post| %>
-  <p><%= post.message %></p>
+  <div id="<%= post.id%>" class="posts">
+    <p><%= post.message %></p>
+    <%= link_to edit_post_path(post.id) do %>
+      Update post
+    <% end %>
+  </div>
+  
```

Comment:
You might be able to use a semantic HTML element here: https://www.w3schools.com/html/html5_semantic_elements.asp